File: library/src/main/java/com/aylax/library/model/AppUsage.java
package com.aylax.library.model;

import android.graphics.drawable.Drawable;

public class AppUsage {
  public Drawable icon;
  public int usage_percentage;
  public String name, pkg_name;
  public long mobile_usage, wifi_usage, total_usage;

  public AppUsage(
      String name,
      String pkg_name,
      Drawable icon,
      int usage_percentage,
      long mobile_usage,
      long wifi_usage,
      long total_usage) {
    this.icon = icon;
    this.name = name;
    this.pkg_name = pkg_name;
    this.wifi_usage = wifi_usage;
    this.total_usage = total_usage;
    this.mobile_usage = mobile_usage;
    this.usage_percentage = usage_percentage;
  }
}

File: library/src/main/java/com/aylax/library/model/AppUsage.kt
package com.aylax.library.model

import android.graphics.drawable.Drawable

data class AppUsage(
    val name: String,
    val pkg_name: String,
    val icon: Drawable?,
    val mobile_usage: Long,
    val wifi_usage: Long,
    val total_usage: Long,
    val usage_percentage: Int
)
